LocalizationUtilities/LocalizationManager.cs

```diff
@@ -72,16 +37,23 @@ private static string GetText(TextAsset textAsset)
 		{
 			sb.AppendLine(byteReader.ReadLine());
 		}
-		return sb.ToString();
+		string str = sb.ToString();
+
+		int startFrom = str.IndexOf(@"{");
+		if (startFrom > 0)
+		{
+			str = str.Substring(startFrom);
+		}
+		return str;
```

Comment:
When I initially made the mod, I didn't know what I was doing, and newer API's were not available. There is a substantially better way to do this.

```cs
const byte leftCurlyBracket = (byte)'{';
byte[] bytes = textAsset.bytes;
int index = Array.IndexOf(bytes, leftCurlyBracket);
if (index < 0)
{
    throw new ArgumentException("TextAsset has no Json content.", nameof(textAsset));
}
return Encoding.UTF8.GetString(new ReadOnlySpan<byte>(bytes, index, bytes.Length - index);
```

Reply:
Done & tested.